functions/examples/injection-tshirt-sizes/image/main.go

```diff
@@ -68,13 +68,12 @@ func size(r *yaml.RNode) error {
 	if size, found := meta.Annotations["tshirt-size"]; !found {
 		// not a tshirt-sized Resource, ignore it
 		return nil
-	} else {
-		// lookup the memory and cpu quantities based on the tshirt size
-		memorySize = memorySizes[size]
-		cpuSize = cpuSizes[size]
-		if memorySize == "" || cpuSize == "" {
-			return fmt.Errorf("unsupported tshirt-size: " + size)
-		}
+	}
+	// lookup the memory and cpu quantities based on the tshirt size
+	memorySize = memorySizes[size]
```

Comment:
The `size` variable is set by the `if`, so this code is invalid after this change (`size` is now referring to this function, whose name the variable shadows)